sklearn/utils/testing.py

```diff
@@ -1024,21 +1015,3 @@ def assert_run_python_script(source_code, timeout=60):
                                % e.output.decode('utf-8'))
     finally:
         os.unlink(source_file)
-
-
-def close_figure(fig=None):
```

Comment:
I'm happy to see this go, but this is public and it's going to be in 0.21...

:s

Reply:
Maybe we could squeeze it in 0.21 ping @jnothman

Reply:
If it is only introduced in 0.21, let's make it private before final release.

Reply:
I can't code right now, but will be happy to approve a PR which does

Reply:
Why no just merging this PR and remove this function directly?

Reply:
If this is merged before release then fine. But in case it is not, a quick fix would do

Reply:
I think that we converged toward a concensus on what to implement. I can quickly answer to review so if anyone have some additional changes to bring, I'll be happy to make them.

Reply:
yes in another pr